examples/manipulator/children.py

```diff
@@ -42,7 +41,22 @@
 JOINT_OFFSET_G = [0, 0, 0]
 
 
+def create_joint_rectangle(x, y, z):
+    geometry = BoxGeometry(x, y, z)
+    for i in range(len(geometry.vertices)):
+        geometry.vertices[i][0] += x / 2
```

Comment:
Untested suggestion:
```suggestion
    for vertices in geometry.vertices:
        vertices[0] += x / 2
```

Reply:
done